ocaml/idl/datamodel_common.ml

```diff
@@ -8,7 +8,7 @@ open Datamodel_roles
               When introducing a new release, bump the schema minor version to the next hundred
               to leave a gap for potential hotfixes needing to increment the schema version.*)
 let schema_major_vsn = 5
-let schema_minor_vsn = 504
+let schema_minor_vsn = 505
```

Comment:
Can you bump this to next hundred like the comment says?

Reply:
I didn't bump to the next hundred because I saw that the latest release further down was 4xx, now I see that quebec is not on the list of releases here, it should be added as well.